test/resources/mnist/checkpoint_every_step_mnist.py

```diff
@@ -0,0 +1,163 @@
+"""Convolutional Neural Network Estimator for MNIST, built with tf.layers."""
```

Comment:
This file seems to be copy/paste of https://github.com/aws/sagemaker-tensorflow-container/blob/script-mode/test/resources/mnist/distributed_mnist.py, is that right?

Should we use the same file but only changing script args between the tests? IMO that is better because of the following:
- easier to update only one model when versions change
- set's a better pattern for our customers to follow

Reply:
It's not exactly the same I had to add some extra configs but those can be parameterized. I will fix it.